src/components/Home/Home.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from "react";
+import { UserContext } from '../../UserContext';
+import { login } from "../Login/Login";
+import { gql, useQuery } from '@apollo/client';
+
+const GET_USER = gql`
+  {
+    getUser(id: "1") {
+      username
+    }
+  }
+`;
+
+const Home = () => {
+  const { user, setUser } = useContext(UserContext);
+  const { loading, error, data } = useQuery(GET_USER);
```

Comment:
Was hoping to only call this query IF there's a user but wasn't sure where this line could live. I tried adding it in the if user conditional but that caused errors living within a return. I was also hoping to pass user.id dynamically to the query but for now hardcoded on line 8.